jams/eval.py

```diff
@@ -475,3 +476,47 @@ def pattern(ref, est, **kwargs):
     est_patterns = pattern_to_mireval(est)
 
     return mir_eval.pattern.evaluate(ref_patterns, est_patterns, **kwargs)
+
+
+def transcription(ref, est, **kwargs):
+    r'''Note transcription evaluation
+
+    Parameters
+    ----------
+    ref : jams.Annotation
+        Reference annotation object
+    est : jams.Annotation
+        Estimated annotation object
+    kwargs
+        Additional keyword arguments
+
+    Returns
+    -------
+    scores : dict
+        Dictionary of scores, where the key is the metric name (str) and
+        the value is the (float) score achieved.
+
+    See Also
+    --------
+    mir_eval.transcription.evaluate
+
+    Examples
+    --------
+    >>> # Load in the JAMS objects
+    >>> ref_jam = jams.load('reference.jams')
+    >>> est_jam = jams.load('estimated.jams')
+    >>> # Select the first relevant annotations
+    >>> ref_ann = ref_jam.search(namespace='pitch_hz')[0]
+    >>> est_ann = est_jam.search(namespace='pitch_hz')[0]
+    >>> scores = jams.eval.transcription(ref_ann, est_ann)
+    '''
+
+    namespace = 'pitch_hz'
```

Comment:
Small note: travis will fail until we merge the ``transcription`` branch of ``mir_eval`` back into ``master`` (because right now it thinks there's no transcription module).